config/core/resources/channel.yaml

```diff
@@ -44,89 +44,62 @@ spec:
       jsonPath: ".status.conditions[?(@.type==\"Ready\")].reason"
     schema:
       openAPIV3Schema:
-        description: 'Channel represents a generic Channel. It is normally used when we want a Channel, but don''t need a specific Channel implementation.'
+        description: 'Channel represents a generic Channel. It is normally used when we want a Channel, but do not need a specific Channel implementation.'
```

Comment:
looks like this is manual change?

wanna update this here too: https://github.com/knative/eventing/blob/main/pkg/apis/messaging/v1/channel_types.go#L32  ?